packages/contracts/deploy-l2/000-OVM_L2StandardTokenFactory.deploy.ts

```diff
@@ -0,0 +1,31 @@
+/* Imports: External */
+import { DeployFunction } from 'hardhat-deploy/dist/types'
+
+/* Imports: Internal */
+import { getContractDefinition } from '../src'
+
+const deployFn: DeployFunction = async (hre: any) => {
+  const { deployments, getNamedAccounts } = hre
+  const { deploy } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const l2TokenFactory = getContractDefinition('OVM_L2StandardTokenFactory', true)
+
+  const factoryOwner = (hre as any).deployConfig.ovmSequencerAddress
```

Comment:
We should be able to delete `factoryOwner`, `initialGasPrice` and the `if` check for them being defined